### DISCUSSION TITLE
v1.0.0 - A total Rewrite

### DISCUSSION THREAD

**Discussion by HandyHat** [HandyHat/ha-hildebrandglow-dcc#284] [Announcements]
This release is a total rewrite of the entire integration to follow Home Assistant best practices, switch to using a library, implement some of the ideas in https://github.com/HandyHat/ha-hildebrandglow-dcc/discussions/191 to reduce strain on Hildebrand's servers and improve the stability of the integration dramatically.

## Breaking Changes

The yearly sensors have been removed. Please use the daily sensors in the energy dashboard, as these now contain the missing 30 minutes at the end of each day.

## Other changes

The standing charge and rate sensors are now disabled by default for new installations as they are used less commonly and many suppliers neglect to provide it. If you do not use these sensors, please disable them to help reduce load on the API.

## Thanks

With many thanks to the Hildebrand Glow team and the contributors to pyglowmarkt, which this integration is now using.

**Full Changelog**: https://github.com/HandyHat/ha-hildebrandglow-dcc/compare/v0.6.0...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/HandyHat/ha-hildebrandglow-dcc/releases/tag/v1.0.0'>v1.0.0 - A total Rewrite</a>.</em>

**Comment by Vincent-Brazil** [comment]
Thanks, so far it doesn't appear in HACS for me, either searching manually or clicking the install link via GitHub.

Had to add as a custom repository in HACS

**Reply by harveyorg** [reply]
same here: follow the instructions on main page for manually adding the integration to HACS.

**Reply by HandyHat** [reply]
Interesting - perhaps it needs to be a HACS default repository for that button to work?

For now I've removed the my link from the readme
